### DISCUSSION TITLE
Where can I learn more about using git?

### DISCUSSION THREAD

**Discussion by PPartisan** [PPartisan/Roky#14] [Q&A]
This project uses `git` as its "Version Control System". Where can I learn more about `git` and how to use it? Do I need to use the command line, or are there graphical tools people recommend? Can I do it with just the IDE?

**Comment by second-ed** [comment]
This is a great source for learning git commands:
https://learngitbranching.js.org/

**Comment by PPartisan** [comment]
This recommendation from Brian on git basics is well-written: https://swcarpentry.github.io/git-novice/.

**Comment by PPartisan** [comment]
[![Git CLI Basics](http://img.youtube.com/vi/CkfF_9mzoPU/0.jpg)](http://www.youtube.com/watch?v=CkfF_9mzoPU "Git CLI Basics")

The git CLI Basics workshop from this morning.

**Comment by stebas101** [comment]
I found this guide useful:
[https://www.atlassian.com/git/tutorials/setting-up-a-repository](https://www.atlassian.com/git/tutorials/setting-up-a-repository)
